doc/user_manual/sampler.tex

```diff
@@ -3285,3 +3294,130 @@ \subsubsection{Metropolis (Metropolis-Hastings Sampler)}
   ...
 </Samplers>
 \end{lstlisting}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%% Adaptive Metropolis Sampler %%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsubsection{Adaptive Metropolis Sampler}
+\label{subsubsubsec:adaptiveMetropolis}
+
+The search for improved proposal distributions of Metropolis sampler is often
+done manually, through trial and error, though this can be difficult especially
+in high dimensions. An alternative approach is adaptive Metropolis, which asks
+the computer to automatically "learn" better parameter values "on the fly".
```

Comment:
"learn" -> ``learn'' 

"on the fly" -> ``on the fly''

(the quotes to be more latex "friendly")

Reply:
fixed